src/easylink/rule.py

```diff
@@ -368,10 +368,10 @@ def get_aggregation_inputs_{self.name}(wildcards):
         raise IncompleteCheckpointException({self.checkpoint_rule_name}.rule, checkpoint_target(output[0]))
     checkpoint_output = glob.glob(f"{{{self.checkpoint_rule_name}.get(**wildcards).output.output_dir}}/*/")
     chunks = [Path(filepath).parts[-1] for filepath in checkpoint_output]
-    return expand(
-        "{self.input_files}",
-        chunk=chunks
-    )"""
+    input_files = []
+    for filepath in {self.input_files}:
```

Comment:
Is this injecting the string representation of a list? Maybe fine but could be a bit more explicit

Reply:
No, it's actually a list of strs.

Reply:
I _hate_ the python integration into snakemake...

Reply:
It's a list, until you interpolate it into this string, right? That's why I'm saying this is injecting the `str(list)` representation